applicationconfig/client/src/main/java/com/azure/applicationconfig/ConfigurationClient.java

```diff
@@ -415,33 +404,33 @@ public Builder httpClient(HttpClient client) {
 
     /**
      * Gets all ConfigurationSetting settings given the {@code nextPageLink} that was retrieved from a call to
-     * {@link ConfigurationClient#listKeyValues(RequestOptions)} or {@link ConfigurationClient#listKeyValues(String)}.
+     * {@link ConfigurationClient#listSettings(RequestOptions)} or a call from this method.
      *
      * @param nextPageLink The {@link Page#nextPageLink()} from a previous, successful call to one of the list operations.
      * @return A stream of {@link ConfigurationSetting} from the next page of results.
      */
-    private Flux<ConfigurationSetting> listKeyValues(@NonNull String nextPageLink) {
+    private Flux<ConfigurationSetting> listSettings(@NonNull String nextPageLink) {
```

Comment:
Given this is a private method, the JavaDoc is unnecessary (you can write it with `/*` instead of `/**`)